src/internal/drag-handle/styles.scss

```diff
@@ -7,16 +7,13 @@
 
 .handle {
   cursor: grab;
+  @include shared.handle();
 }
 
 .handle:active {
   cursor: grabbing;
 }
 
-.handle:not(.active):focus-visible {
-  @include shared.focus-highlight();
-}
-
```

Comment:
The focus style is handled by the `InternalDragHandle` from the components package.